examples/Demo/FluentUI.Demo.Client/Documentation/Components/Spacer/Examples/SpacerDefault.razor

```diff
@@ -0,0 +1,9 @@
+﻿<div style="display: flex; width=100%">
```

Comment:
Can you use the FluentUI components :-) => `FluentStack`

Reply:
And the style `width=100%` is incorrect: (1) not an `=` symbol and missing `;` and the end.